<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUserManageDialog</class>
 <widget class="QDialog" name="CUserManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_edit_user">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>98</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modify_pwd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>98</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pre_page">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return_back">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>340</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
